src/Services/ConvertToLearnosityService.php

```diff
@@ -369,22 +382,22 @@ private function getTaxonPathEntryForItemTags(\DOMNode $resource)
      *
      * @return array - the results of the conversion
     */
-    private function convertAssessmentItemInFile($contents, $itemReference = null, array $metadata = [], $currentDir, $resourceHref, $itemTagsArray = [])
+    private function convertAssessmentItemInFile($contents, $itemReference = null, array $metadata = [], $currentDir, $resourceHref, $itemTagsArray = [], $resourceType)
     {
         $results = null;
 
         try {
             $xmlString = $contents;
+
             // Check that we're on an <assessmentItem>
-            if (!CheckValidQti::isAssessmentItem($xmlString)) {
+            if ($resourceType == static::RESOURCE_TYPE_ITEM && !CheckValidQti::isAssessmentItem($xmlString)) {
                 $this->output->writeln("<info>" . static::INFO_OUTPUT_PREFIX . "Not an <assessmentItem>, moving on...</info>");
                 return $results;
             }
 
             $resourcePath = $currentDir . '/' . $resourceHref;
-            $results = $this->convertAssessmentItem($xmlString, $itemReference, $resourcePath, $metadata, $itemTagsArray);
-
-        } catch (\Exception $e) {
+            $results = $this->convertAssessmentItem($xmlString, $itemReference, $resourcePath, $metadata, $itemTagsArray, $resourceType);
+            } catch (\Exception $e) {
```

Comment:
Could you please move backwards in order to align the indent?

Reply:
Done